package.json

```diff
@@ -49,5 +49,10 @@
     "expand-tilde": "^2.0.2",
     "superagent": "^3.5.0",
     "uuid": "^3.0.1"
+  },
+  "config": {
+    "blanket": {
+      "pattern": "xlsx.js"
+    }
```

Comment:
Are you using this for something? Jest has a coverage option, and according to the repo for blanket, it is no longer actively maintained. So if you're not using it I think you can remove it.